readme.md

```diff
@@ -40,124 +40,139 @@ $ npm -v
+npm ERR! Test failed. See above for more details.
+
+test Scripts from package.json are run
 
 - [ ] Create a new "script" command called "start" that has the following value: `node index.js`
 
 * **Question:** What will you enter on the command line to run that script?
 
 * **Your Answer:**
 
----
+---npm start
 
 - [ ] Change the name of your "start" script to "my-file"
 
 * **Question:** The same pattern will not work to try and run this script. How can you successfully get this script to run?
 
 * **Your Answer:**
 
----
+---npm run my-file
 
 - [ ] Create a new file called `profile.js`. Inside the file, copy the following but replace `<your-name>` with your name:
+
   ```js
-  module.exports = '<your-name>'
+  module.exports = "<your-name>";
   ```
 
   Add the following to your `index.js` file. Then, run your file.
+
   ```js
-  const profile = require('./profile.js')
-  console.log(profile)
+  const profile = require("./profile.js");
+  console.log(profile);
   ```
 
 * **Question:** What gets logged? Why?
 
 * **Your Answer:**
+  Hello, Node!
+  Rubini
+  The file profile.js is logged in the console.
 
-* **Question:** What is `module.exports` and what is its _type_ in JavaScript? What is `require` and what is its _type_ in JavaScript?
+- **Question:** What is `module.exports` and what is its _type_ in JavaScript? What is `require` and what is its _type_ in JavaScript?
 
-* **Your Answer:**
+- **Your Answer:**
 
----
+---'module.exports' is the object that's actually returned as the result of a require call - it exposes internally scoped things out of module. Here the type is string.
+'require' is used to load to the module which to be exposed. Here, the type is string.
 
 - [ ] We can only export one thing from files when using Node. With that said, export both your name and your birthday from the `profile.js` file.
 
 * **Question:** What are some ways you can solve this problem?
 
 * **Your Answer:**
 
----
+---Wrap them in a collection like object or array and access them. Thus export only one object.
 
 - [ ] Add the following to your `index.js` file. Then, run your file.
   ```js
-  const path = require('path')
-  console.log(path.resolve())
+  const path = require("path");
+  console.log(path.resolve());
   ```
 
 * **Question:** What is `path` and where does it come from?
 
 * **Your Answer:**
 
----
+---node gets the path(the folder we are currently in) from the built in module Path. The path.resolve() method resolves a sequence of paths or path segments into an absolute path.
 
 - [ ] Install the [moment](https://www.npmjs.com/package/moment) package
 
 * **Question:** What command can you run to install this package?
 
 * **Your Answer:**
 
----
+---npm install moment
 
 - [ ] On your own, use this package in the `index.js` file
 
 * **Question:** Do you need to use a `./` to require the package? Why or why not?
 
 * **Your Answer:**
 
----
+---No. It depends on the where the file or folder is placed inside the package.
```

Comment:
This question is asking about why you wouldn't do the following:
```js
const moment = require('./moment')
```

So, in this case you do not do the above because you do not have a _file_ called moment, but a package you've installed.